### PR TITLE
Jk se/pi 621/node lib updates

### PR DESCRIPTION
**Pull request by jkraut0517**

### CODE REVIEW

File: src/models/configuration/settings.model.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+'use strict';
+
+
+import { ItemSenseEndpoint } from '../itemsense-endpoint.model.js';
+
+export class Settings extends ItemSenseEndpoint{
+
+  constructor() {
+    let path = 'configuration/v1/settings';
+    let endpointInfo = { path: path };
```
Comment (4141done) [inline src/models/configuration/settings.model.js]: could be { path };

File: src/models/configuration/settings.model.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+'use strict';
+
+
+import { ItemSenseEndpoint } from '../itemsense-endpoint.model.js';
+
+export class Settings extends ItemSenseEndpoint{
+
+  constructor() {
+    let path = 'configuration/v1/settings';
```
Comment (4141done) [inline src/models/configuration/settings.model.js]: Both of these should be ```const```

**Comment by jkraut0517** [comment]
@4141done changes have been made as per your request.

**Comment by jkraut0517** [comment]
@4141done the README has been updated!

**Comment by seansellek** [comment]
Package is now ready for publishing alongside the next platform release

**Pull request merged**